hub/api/v1/sql.py

```diff
@@ -77,31 +79,36 @@ class VectorStoreFile(BaseModel):
 
 class SqlClient:
     def __init__(self):  # noqa: D107
-        self.db = pymysql.connect(
+        # save initial value for db autocommit property
+        self.db_autocommit = getenv("DATABASE_AUTOCOMMIT", "true").lower() == "true"
+
+        self.db = psycopg.connect(
             host=getenv("DATABASE_HOST"),
             user=getenv("DATABASE_USER"),
             password=getenv("DATABASE_PASSWORD"),
-            database=getenv("DATABASE_NAME"),
-            autocommit=True,
+            dbname=getenv("DATABASE_NAME"),
+            autocommit=self.db_autocommit,
+            options="-c search_path=" + getenv("DATABASE_SCHEMA"),
```

Comment:
If this is public can we add a default?
```suggestion
            options="-c search_path=" + getenv("DATABASE_SCHEMA", "public"),
```